inner-loop-scripts/shell/1-cluster-stamp.sh

```diff
@@ -72,32 +72,14 @@ kubectl create namespace cluster-baseline-settings
 kubectl apply -f ../../cluster-manifests/cluster-baseline-settings/flux.yaml
 kubectl wait --namespace cluster-baseline-settings --for=condition=ready pod --selector=app.kubernetes.io/name=flux --timeout=90s
 
+ACR_NAME=$(az deployment group show -g $RGNAMECLUSTER -n cluster-0001 --query properties.outputs.containerRegistryName.value -o tsv)
+# Import ingress controller image hosted in public container registries
+az acr import --source docker.io/library/traefik:2.2.1 -n $ACR_NAME
```

Comment:
We need to do this in the instructions as well, right?

Reply:
No, it is there and it was not here
https://github.com/mspnp/aks-secure-baseline/blob/main/08-secret-managment-and-ingress-controller.md
point 5.

Reply:
Ah, got it.